tasks/kibana.yml

```diff
@@ -1,15 +1,23 @@
 ---
-- name: Download Kibana tarball.
-  get_url:
-    url: https://download.elastic.co/kibana/kibana/kibana-{{ elk_kibana_version }}-linux-x64.tar.gz
-    dest: /opt/kibana-{{ elk_kibana_version }}-linux-x64.tar.gz
-    # Don't download file every time, only if it doesn't exist.
-    force: no
+- name: Add Kibana repo apt key.
+  apt_key:
+    url: https://packages.elastic.co/GPG-KEY-elasticsearch
```

Comment:
I wanna note that this is the same key used for ES, so in the context of the whole role it's a duplicate task. Remove?